Realm/RLMAccessor.h

```diff
@@ -17,10 +17,11 @@
 ////////////////////////////////////////////////////////////////////////////
 
 #import <Foundation/Foundation.h>
-#import "RLMObjectStore.hpp"
 
 @class RLMObjectSchema, RLMProperty, RLMObjectBase;
 
+typedef NSUInteger RLMCreationOptions;
```

Comment:
This should be moved to RLMConstants rather than using a forward/re-declaration.